bitnami/wavefront-adapter-for-istio/Chart.yaml

```
@@ -0,0 +1,33 @@
+annotations:
+  category: Infrastructure
+apiVersion: v2
+appVersion: 1.0.3
+dependencies:
+  - condition: wavefront.enabled
+    name: wavefront
+    repository: https://charts.bitnami.com/bitnami
+    version: 1.x.x
```

There is a new major version
```suggestion
    version: 2.x.x
```